projects/igniteui-angular/src/lib/splitter/splitter-pane/splitter-pane.component.ts

```diff
@@ -216,7 +201,6 @@ export class IgxSplitterPaneComponent {
         // reset sibling sizes when pane collapse state changes.
         this._getSiblings().forEach(sibling => sibling.size = 'auto');
         this.collapsed = !this.collapsed;
-        this.onToggle.emit(this);
         this.collapsedChange.emit(this);
```

Comment:
Shouldn't collapse change be emitted with the state of the collapsed property, instead of with `this`?

Reply:
True, we'll have to deprecate onToggle then as this cannot be migrated.